rclcpp_action/test/test_client.cpp

```diff
@@ -273,12 +273,14 @@ TEST_F(TestClient, async_send_goal_but_ignore_feedback_and_result)
   auto action_client = rclcpp_action::create_client<ActionType>(client_node, action_name);
   ASSERT_TRUE(action_client->wait_for_action_server(WAIT_FOR_SERVER_TIMEOUT));
 
+  printf("Testing Bad Goal...\n");
```

Comment:
The `printf`'s in this file seem unnecessary, or at least are unrelated changes.

Reply:
Yeah, I was doing some debugging on that and forgot to get rid of them.